index.rst

```diff
@@ -87,6 +87,57 @@ Catalog Data Access
 We assume that most user analysis cases that involve spatially localized searches of the Object or Source table, and/or the retrieval of ForcedSource data for limited sets of Objects, will go to Qserv via the TAP service.
 
 
+Bulk reprocessing of commissioning data
+---------------------------------------
+
+During Rubin/LSST commissioning it will be necessary to reprocess the data collected to date with evolving versions of the pipelines. 
+
+
+Processing of simulated images
+------------------------------
+
+Many analyses will benefit from being able to compare the LSST data products with the same analysis run on simulated images or catalogs. 
```

Comment:
This is a great point.  Is this going to be an online project-supported use case, in general - the processing of (PhoSim?) simulated images through the pipeline?

Reply:
Not that I am aware of, so my concern is that users all run their own reprocessing of simulated images.